src/main/java/net/minecraftforge/gradle/common/BasePlugin.java

```diff
@@ -217,13 +214,7 @@ public void afterEvaluate() {
     private boolean hasMavenCentralBeforeJCenterInBuildScriptRepositories() {
         if (ProjectUtils.getBooleanProperty(project, "com.anatawa12.forge.gradle.no-maven-central-warn"))
             return true;
-        URI mavenCentralUrl;
-        try {
-            mavenCentralUrl = project.uri(ArtifactRepositoryContainer.class
-                    .getField("MAVEN_CENTRAL_URL").get(null));
-        } catch (IllegalAccessException | NoSuchFieldException e) {
-            throw new RuntimeException(e);
-        }
+        URI mavenCentralUrl = project.uri(ArtifactRepositoryContainer.MAVEN_CENTRAL_URL);
```

Comment:
I don't see any change in github source code:
[4.4.1](https://github.com/gradle/gradle/blob/v4.4.1/subprojects/core-api/src/main/java/org/gradle/api/artifacts/ArtifactRepositoryContainer.java#L52)
[latest](https://github.com/gradle/gradle/blob/master/subprojects/core-api/src/main/java/org/gradle/api/artifacts/ArtifactRepositoryContainer.java#L52)

Reply:
Currently, 5.0 is the tested minimum Gradle version but this is/was supported to run on around Gradle 4.1 (Gradle version mod template of 1.7.10 uses) and since that version, it was changed.
and according to this fact, this value may be changed in the feature & the change will break mavenCentral detection so I want to get latest value at runtime.